src/Files.App/UserControls/StatusBar.xaml

```diff
@@ -111,17 +111,17 @@
 				<Button.Flyout>
 					<MenuFlyout>
 						<MenuFlyoutItem
-							x:Name="OpenFolderInVSCodeButton"
-							Command="{x:Bind Commands.OpenInVSCode}"
-							KeyboardAcceleratorTextOverride="{x:Bind Commands.OpenInVSCode.HotKeys, Mode=OneWay}"
-							Text="{x:Bind Commands.OpenInVSCode.Label, Mode=OneWay}"
-							ToolTipService.ToolTip="{x:Bind Commands.OpenInVSCode.LabelWithHotKey, Mode=OneWay}" />
+							x:Name="OpenFolderInIDEButton"
```

Comment:
Can you search the entire codebase (including the automation tests) to make sure the old name isn't being referenced anywhere?